portfolio/src/main/webapp/script.js

```diff
@@ -72,21 +87,23 @@ function getComments() {
 
 /* Creates an <li> element containing text. */
 function addComment(comment) {
-  const commentItem = document.createElement("li");
-  const commentHeader = document.createElement("div");
-  const commentName = document.createElement("p");
-  const commentMessage = document.createElement("p");
-  const commentDelete = document.createElement("img");
-
-  commentItem.classList.add("comment");
-  commentHeader.classList.add("comment-header");
-  commentName.classList.add("comment-name");
-  commentMessage.classList.add("comment-message");
-  commentDelete.classList.add("comment-delete");
-
-  commentName.innerText = comment.name;
-  commentMessage.innerText = comment.message;
-  commentDelete.onclick = function() { deleteComment(comment.id); }; 
+  const commentItem = createElementWithParams("li", { className: "comment" })
+  const commentHeader = createElementWithParams("div", { className: "comment-header" })
+
+  const commentName = createElementWithParams("p", { 
+    className: "comment-name", 
+    innerText: comment.name 
```

Comment:
**Really** Nit: Google style guide recommends adding a trailing comma when you have an object literal with line breaks.

See https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma